docs/boards.md

```diff
@@ -37,9 +37,9 @@ This editor supports various maker boards, based on SAMD21, SAMD51, NRF52.
         "imageUrl": "/static/libs/redbear-ble-nano.jpg"  
     },
     {
-        "name": "Add your board",
-        "url": "/boards/add-a-new-board",
-        "imageUrl": "/static/libs/bead-servo.jpg"
+        "name": "Robotics Masters Robo HAT MM1 M4",
```

Comment:
Leave this card and add a new one.

Reply:
Not 100% sure this is fixed correctly.  Feedback required.